packages/apps/tenant/templates/quota.yaml

```
@@ -0,0 +1,27 @@
+{{- if .Values.resourceQuotas }}
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: tenant-quota
+  namespace: {{ include "tenant.name" . }}
+spec:
+  hard:
+    {{- if .Values.resourceQuotas.hard}}
+    {{- if .Values.resourceQuotas.hard.resources.requests }}
+    {{- if .Values.resourceQuotas.hard.resources.requests.cpu }}
+    requests.cpu: {{ .Values.resourceQuotas.hard.resources.requests.cpu }}
+    {{- end }}
+    {{- if .Values.resourceQuotas.hard.resources.requests.memory }}
+    requests.memory: {{ .Values.resourceQuotas.hard.resources.requests.memory }}
+    {{- end }}
+    {{- end }}
+    {{- if .Values.resourceQuotas.hard.resources.limits }}
+    {{- if and .Values.resourceQuotas.hard.resources.limits.cpu }}
+    limits.cpu: {{ .Values.resourceQuotas.hard.resources.limits.cpu }}
+    {{- end }}
+    {{- if .Values.resourceQuotas.hard.resources.limits.memory }}
+    limits.memory: {{ .Values.resourceQuotas.hard.resources.limits.memory }}
+    {{- end }}
+    {{- end }}
+    {{- end }}
```

It might be way more easier to do:

```suggestion
spec:
  hard:
    {{- toYaml .Values.resourceQuotas | nindent 4 }}
```